pac4j-core/src/main/java/org/pac4j/core/util/HttpUtils.java

```diff
@@ -18,9 +18,9 @@
  */
 public final class HttpUtils {
 
-    private static int connectTimeout = HttpConstants.DEFAULT_CONNECT_TIMEOUT;
+    private static int defaultConnectTimeout = HttpConstants.DEFAULT_CONNECT_TIMEOUT;
```

Comment:
I'm not sure to understand why we now have default timeouts instead of timeouts...